tests/src/test/scala/whisk/core/cli/test/ApiGwRestTests.scala

```diff
@@ -263,14 +263,21 @@ class ApiGwRestTests extends ApiGwTests {
       val testRelPath = "/api2/greeting2/{name}"
       val testRelPathGet = "/api2/greeting2/name"
       val hostRegex = "%HOST%".r
+      val namespaceRegex = "%NAMESPACE%".r
       var file = TestUtils.getTestActionFilename(s"echo-web-http.js")
       assetHelper.withCleaner(wsk.action, actionName, confirmDelete = true) { (action, _) =>
         action.create(actionName, Some(file), web = Some("true"))
       }
       try {
         val apiGwURL = "https://" + wskprops.apihost
         file = TestUtils.getTestApiGwFilename("apigw_path_param_support_test_withPathParameters1.json")
-        file = replaceStringInFile(file, hostRegex, apiGwURL)
+        var replacements = Map(hostRegex -> apiGwURL)
+        if (wskprops.apihost.contains("usma.ibm.com")) {
```

Comment:
@jessealva - this test is in open source, so performing special "ibm" checks should not be used.  what issue is trying to be overcome?

also, note that the cli tests have move to the apache/incubator-openwhisk-cli repo.